wp-content/themes/access/controllers/programs.php

```diff
@@ -216,14 +216,25 @@ public function faqAnswer($field) {
   /**
    * Add FAQ to schema only if sections are shown.
    *
-   * @return Boolean weather to add FAQ to schema.
+   * @return Array set of questions to be added to the schema.
    */
-  public function validateFaq() {
+  public function addQuestionsToSchemaFaq($questions) {
```

Comment:
RE: to adding schemas to the controller, you could have one method that just builds the schema in this function.